Orm/Xtensive.Orm/Orm/Linq/Translator.Queryable.cs

```diff
@@ -1263,11 +1263,18 @@ private Expression VisitExistsAsInclude(Expression source, LambdaExpression pred
       var parameter = predicate.Parameters[0];
       ProjectionExpression visitedSource;
       using (state.CreateScope()) {
-        if (source.IsLocalCollection(context) &&
-            (source.Type.IsGenericType && source.Type.GetGenericArguments()[0].IsAssignableFrom(typeof (Key))) ||
-            (source.Type.IsAssignableFrom(typeof (Key)))) {
-          var localCollecctionKeyType = LocalCollectionKeyTypeExtractor.Extract((BinaryExpression)predicate.Body);
-          state.TypeOfEntityStoredInKey = localCollecctionKeyType;
+        if (source.IsLocalCollection(context)) {
+          var sourceType = source.Type;
+          Type itemType = sourceType.IsGenericType

```

Comment:
I think it is better to use `var` for code consistency.

Reply:
Yep. Thank you for mentioning that. There was another problem couple lines down.